README.md

```diff
@@ -1,60 +1,73 @@
-Fluxxed_up
+Fluxxed_up [![Build Status](https://travis-ci.org/everplans/fluxxed_up.svg?branch=master)](https://travis-ci.org/everplans/fluxxed_up) [![Code Climate](https://codeclimate.com/repos/57361f9378bbd61943002b6b/badges/59cae8b81ec09ab798fb/gpa.svg)](https://codeclimate.com/repos/57361f9378bbd61943002b6b/feed) [![Dependency Status](https://gemnasium.com/badges/github.com/everplans/fluxxed_up.svg)](https://gemnasium.com/github.com/everplans/fluxxed_up)
```

Comment:
This PR is branched from [my previous PR](https://github.com/everplans/fluxxed_up/pull/47), so that should be reviewed and merged first